pkgs/data/fonts/fsmrmp/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchgit
+, fetchurl
+, pkgs
```

```suggestion
, python3
, fontforge
, unar
```
Do not use `pkgs` as a parameter. This is already being called by `callPackage`, so you will only need to add the needed packages.